### ISSUE TITLE
RSS Feed Not Showing File Size for AnimeTosho

### ISSUE THREAD

**Issue by sdburrows** [nzbgetcom/nzbget#307]
### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### NZBGet Version

v24.1-stable

### Platform

All

### Environment

```markdown
Device: Windows 11
```

### Current Behavior

NZBGet RSS feed not showing file size when connected to Anime Tosho RSS link: https://feed.animetosho.org/rss2

### Expected Behavior

Show file size

**Comment by dnzbk** [comment]
Confirmed. I'll take a look.

**Comment by dnzbk** [comment]
Couldn't confirm, sorry.
At least 
_Kinnikuman Perfect Origin Arc S01E00 1080p NF WEB-DL AAC2 0 H 264-VARYG (Kinnikuman Kanpeki Chоujin Shiso-hen Episode 0, Multi-Subs)_
works well. Total size 988 MB.
And I can see the full and remaining sizes while downloading:
![image](https://github.com/nzbgetcom/nzbget/assets/146707790/a61597fd-307f-48f5-aa45-b12fd7c63a4a)

**Comment by sdburrows** [comment]
@dnzbk I should have been clearer, in the feed preview, there is no size. I do RSS filtering by size and there is no size
![Screenshot 2024-07-08 195144](https://github.com/nzbgetcom/nzbget/assets/50125721/2e0e0ec1-d04b-4a8f-9a37-74930a97541f)

**Comment by dnzbk** [comment]
@sdburrows 
Yeah, I got it now. I'll debug it. 
Thank you for spotting this.

**Comment by sdburrows** [comment]
@dnzbk Thanks. I am also receiving the following errors, not sure if they are related, but the download went ahead anyway.

```
ERROR	Mon Jul 08 2024 07:11:02	Could not add collection 2222.nzb to queue
ERROR	Mon Jul 08 2024 07:11:02	Error parsing nzb-file 2222.nzb
ERROR	Mon Jul 08 2024 07:11:02	Error parsing nzb-file: Start tag expected, '<' not found
ERROR	Mon Jul 08 2024 07:11:00	Could not add collection 1111.nzb to queue
ERROR	Mon Jul 08 2024 07:11:00	Error parsing nzb-file 1111.nzb
```

**Comment by dnzbk** [comment]
@sdburrows 
Yeah, I noticed the same errors yesterday when I was testing this feed.
We'll take a look.
